deploy-service/common/src/main/java/com/pinterest/deployservice/handler/CommonHandler.java

```diff
@@ -398,10 +398,12 @@ public void transitionDeployState(String deployId, EnvironBean envBean) throws E
         String lockName = String.format("STATE_TRANSITION-%s", deployId);
         Connection connection = utilDAO.getLock(lockName);
         if (connection != null) {
+            LOG.debug(String.format("Successfully get lock: %s", lockName));
             try {
                 internalTransition(deployId, envBean);
             } finally {
                 utilDAO.releaseLock(lockName, connection);
+                LOG.debug(String.format("Successfully released lock: %s", lockName));
```

Comment:
maybe we can use a more "grep/searchable"-able msg for check for lock leak, something like "Lock Successfully released or get" in case the search does not support regex

Reply:
these messages can be searched by "Successfully released lock", "Successfully get lock".

Reply:
it is better to search by one filter so you can see the get/release pair next to either to identify the leak.

Reply:
i see what you mean. will make the change.